modules/yaml

```diff
@@ -0,0 +1 @@
+/home/aireilly/openshift-docs/yaml
```

Comment:
I think this symlink is broken and should be something more like `../yaml`

![image](https://user-images.githubusercontent.com/32749632/203839818-57b83c5c-bce5-4c32-a806-8138a90bcaef.png)

Reply:
derp